OpenShift/Standard/serviceaccelerator.md

```diff
@@ -0,0 +1,560 @@
+# Detailed Security Configuration
```

Comment:
Does this contain the same content as the asciidoc version?  I skimmed it and it seems similar - if so we should delete one and keep one.